### ISSUE TITLE
Argoverse Dataset Bucket Does Not Exist

### ISSUE THREAD

**Issue by jeffreyianwilson** [ultralytics/ultralytics#3948]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

PyTorch Hub

### Bug

Attempts to train Argoverse dataset returns an error that the bucket does not exist.
https://argoverse-hd.s3.us-east-2.amazonaws.com/Argoverse-HD-Full.zip

This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>NoSuchBucket</Code>
<Message>The specified bucket does not exist</Message>
<BucketName>argoverse-hd</BucketName>
<RequestId>PBPFK1EQRGK4PVHW</RequestId>
<HostId>RmB+ZO3tkjyga24hSbNGIMJ/ZD/Gii5uQWvc/mH/Z8m2l0wd4EjBunf/QGuI3bzAxwt9dZ+4xPQ=</HostId>
</Error>

### Environment

Ultralytics Hub Train, local agent

### Minimal Reproducible Example

from ultralytics import YOLO, checks, hub

checks()
hub.login('myapi')

model = YOLO('https://hub.ultralytics.com/models/7SU8EdZVJNWVT0XBDYus')
model.train()

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by KetanB17395** [comment]
same issue occurs while trying to fetch argovers dataset from this https://argoverse-hd.s3.us-east-2.amazonaws.com/Argoverse-HD-Full.zip

**Comment by Burhan-Q** [comment]
Yes, this is the one. That's odd, their [site](https://www.argoverse.org/av1.html#download-link) still has (what looks like) a lot of the data available to download, but it's broken up into multiple `tars` instead of a single `zip`

**Comment by KetanB17395** [comment]
https://www.kaggle.com/datasets/mtlics/argoversehd  thanks for this link

**Comment by KetanB17395** [comment]
@glenn-jocher sure and thanx

**Comment by glenn-jocher** [comment]
@KetanB17395 You're welcome! I'm glad we could help. If you have any additional questions or run into any other issues, please feel free to ask. Happy coding with YOLOv8!

**Comment by jeffreyianwilson** [comment]
If it's not compatible, why is it on the Ultralytics Hub?

**Comment by glenn-jocher** [comment]
@KetanB17395 i apologize for the confusion. The inclusion of the Argoverse dataset on the Ultralytics Hub might have been an oversight, as YOLOv8 model currently does not natively support 3D-Tracklet and LiDAR data formats, which are the formats used by Argoverse. Our team will be reviewing and updating the hub to ensure that all listed datasets are compatible with YOLOv8. Thank you for bringing this to our attention.

**Comment by glenn-jocher** [comment]
@jeffreyianwilson ah yes, we do support Argoverse for object detection, so untill recently you could autodownload and start training directly on the dataset, i.e. `yolo train model=yolov8n.pt data=ArgoverseHD.yaml`. 

The problem is the originally hosted download zip was removed after Ford shut down Argo AI, their self-driving aquisition/arm.

**Comment by glenn-jocher** [comment]
@jeffreyianwilson great news 😃! Your original issue may now be partially resolved ✅ in PR #4196. I've uploaded the Argoverse-HD-Full.zip file that is required for Argoverse training to our Google Drive at https://drive.google.com/file/d/1st9qW3BeIwQsnR0t8mRpvbsSWIo16ACi/view?usp=drive_link

You can download this file manually and place it in your CWD when you run the `yolo train data=Argoverse.yaml` command and it should be found and unzipped correctly.

The PR also adds autodownload capability but I've seen that Google places quite strict quotas on automatically downloading from Drive, so you'll probably initially get a Quota Exceeded error prompting you to download the file manually.

To get this update:
- **[Git](https://github.com/ultralytics/ultralytics)** – Run `git pull` from within your `ultralytics/` directory or run `git clone https://github.com/ultralytics/ultralytics` again
- **[Pip](https://pypi.org/project/ultralytics/)** – Update with `pip install -U ultralytics`
- **[Notebooks](https://github.com/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb)** – Check out the updated notebooks  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/ultralytics)** – Run `sudo docker pull ultralytics/ultralytics:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and letting us know. Please confirm if this update fixes the issue for you, and don't hesitate to report any other **issues** you find or **feature requests** you may have. Happy training with YOLOv8 🚀!
